RxBlocking/BlockingObservable+Operators.swift

```diff
@@ -197,4 +197,64 @@ extension BlockingObservable {
 
         return element
     }
+
+    /**
+     Blocks current thread until sequence terminates.
+     
+     If sequence terminates with error before producing first element, terminating error will be thrown.
+     
+     - parameter predicate: A function to test each source element for a condition.
+     - returns: Returns the only element of an sequence that satisfies the condition in the predicate, and reports an error if there is not exactly one element in the sequence.
+     */
+    public func single(predicate: (E) throws -> Bool) throws -> E? {
```

Comment:
It would be great if you could just make `single` version without predicate be an alias for this one 

```
public func single() throws -> E? {
    return single { _ in true }
}
```

I think that in this case there aren't any reasons why we would want to keep that code almost duplicated.